code/game/gamemodes/dynamic/dynamic_rulesets.dm

```diff
@@ -84,13 +84,13 @@
 	var/logged_repeated_mode_adjust = FALSE
 
 
-/datum/dynamic_ruleset/New()
+/datum/dynamic_ruleset/New(datum/game_mode/dynamic/dynamic_mode)
 	// Rulesets can be instantiated more than once, such as when an admin clicks
 	// "Execute Midround Ruleset". Thus, it would be wrong to perform any
 	// side effects here. Dynamic rulesets should be stateless anyway.
 	SHOULD_NOT_OVERRIDE(TRUE)
 
-	mode = SSticker.mode
+	mode = dynamic_mode
```

Comment:
what's this about? was the mode not getting set?

Reply:
The gamemode was being set to the wrong thing, it was being set to the global gamemode with 0 players instead of the dynamic simulation gamemode it creates